docs/src/pages/components/AriaLive/WebAriaLivePage.js

```diff
@@ -0,0 +1,288 @@
+          <Paragraph>
+            ARIA live region with <BpkCode>id=&quot;{this.id}&quot;</BpkCode>.
+            This content is relevant to everyone, not just assistive
+            technologies, so it is permanently visible.
+          </Paragraph>
+          <Paragraph>
+            <strong>
+              Searching for {direct ? 'direct ' : ''}flights to {destination}.
+            </strong>
+          </Paragraph>
+        </BpkAriaLive>
+      </div>
+    );
+  }
+}
+
+class ChipsExample extends React.Component<
+  {},
+  {
+    categories: { Flights: boolean, Hotels: boolean, 'Car hire': boolean },
+    updates: Array<string>,
+  },
+> {
+  constructor() {
+    super();
+    this.state = {
+      categories: {
+        Flights: true,
+        Hotels: true,
+        'Car hire': false,
+      },
+      updates: [],
+    };
+  }
+
+  id = 'aria-live-chips-example';
+
+  toggleCategory = category => {
+    this.setState(prevState => {
+      const nextState = prevState;
+      nextState.categories[category] = !prevState.categories[category];
+      nextState.updates.push(
+        `${category} became ${
+          nextState.categories[category] ? 'enabled' : 'disabled'
+        }.`,
+      );
+      return nextState;
+    });
+  };
+
+  render() {
+    const { categories, updates } = this.state;
+    return (
+      <div>
+        <div className={getClassName('bpkdocs-aria-live-page__chips-wrapper')}>
+          <Paragraph>
+            Interactive component with
+            <BpkCode>aria-controls=&quot;{this.id}&quot;</BpkCode> to link it to
+            the ARIA live region below with the same ID.
+          </Paragraph>
+          {Object.keys(categories).map(category => (
+            <BpkChip
+              className={getClassName('bpkdocs-aria-live-page__chip')}
+              aria-controls={this.id}
+              selected={categories[category]}
+              accessibilityLabel={category}
+              onClick={() => {
+                this.toggleCategory(category);
+              }}
+            >
+              {category}
+            </BpkChip>
+          ))}
+        </div>
+        <BpkAriaLive
+          visible
+          id={this.id}
+          className={getClassName('bpkdocs-aria-live-page__aria-live')}
+        >
+          <Paragraph>
+            ARIA live region with <BpkCode>id=&quot;{this.id}&quot;</BpkCode>.
+            Ordinarily this would be only visible to assistive technologies.
+          </Paragraph>
+          <Paragraph>
+            As the above chips are toggled, updates appear here.
+          </Paragraph>
+          {updates.map(update => (
+            <Paragraph>
+              <strong>{update}</strong>
+            </Paragraph>
+          ))}
+        </BpkAriaLive>
+      </div>
+    );
+  }
+}
+
+const components = [
+  {
+    id: 'about',
+    title: 'About ARIA live regions',
+    blurb: [
+      <Paragraph>
+        Oftentimes, pages update without the content reloading completely. For
```

Comment:
Is oftentimes a word?

Reply:
Indubitably, but I'll change it for something more accessible